### PR TITLE
Added pandas error

### PR DESCRIPTION
**Pull request by eliotwrobson**
@coatless as requested, for issue #6501.

### CODE REVIEW

**Comment by coatless** [comment]
@eliotwrobson thanks! Glancing at the file change, I'm only seeing a single problem get modified. The routing change needs to happen inside of `python/prairielearn.py` with `dataframe-v2`. Otherwise, the old serialization technique is being used which isn't JSON-serializable.

**Comment by eliotwrobson** [comment]
@coatless yes, that change will need to be in the `to_json` / `from_json` files. But the question code that I modified is a minimal example showing the exception that gets raised by using the builtin pandas json encoding functions.

File: exampleCourse/questions/element/pythonVariable/server.py [hunk 1]
```diff
@@ -8,6 +8,10 @@
 
 def generate(data):
     df = pd.io.parsers.read_csv("breast-cancer-train.dat", header=None)
+
+    encoded_json_df = df.to_json(orient = "table", date_format = "iso")
+    pd.read_json(encoded_json_df, orient="table")
```
[the remark this answers is not in the document]
Comment (coatless) [reply exampleCourse/questions/element/pythonVariable/server.py:13]: @eliotwrobson that should work.

[the remark this answers is not in the document]
Comment (eliotwrobson) [reply exampleCourse/questions/element/pythonVariable/server.py:13]: Tried and still running into problems. Will continue discussion on main issue.

**Pull request closed without merging**